huaweicloud/services/mpc/resource_huaweicloud_mpc_transcoding_template.go

```diff
@@ -0,0 +1,503 @@
+package mpc
+
+import (
+	"context"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	mpc "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/mpc/v1/model"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func ResourceMPCTranscodingTemplate() *schema.Resource {
```

Comment:
`MPC` is redundant.

Reply:
There is service name in most of our resource functions. I don't think this need to be change.

Reply:
I disagree, you will specify `mpc` twice if you refernece in the other packages: `mpc.ResourceMPCTranscodingTemplate()`.

Reply:
fair enough